api4/channel_test.go

```diff
@@ -860,7 +859,7 @@ func TestDeleteChannel(t *testing.T) {
 
 	// successful delete of channel with multiple members
 	publicChannel3 := th.CreatePublicChannel()
-	th.App.AddUserToChannel(user2, publicChannel3)
+	th.App.AddUserToChannel(user, publicChannel3)
```

Comment:
It looks like we are changing the test here. Is not the exact same behavior. I'm not sure if this changes are correct.

Reply:
You are correct, actually the test was before incorrect and lacked a single `AddUserToChannel` call for `user`. I fixed it accordingly. Thank you!